app/javascript/packages/validated-field/validated-field-element.ts

```diff
@@ -69,11 +70,8 @@ class ValidatedFieldElement extends HTMLElement {
   setErrorMessage(message?: string | null) {
     if (message) {
       this.getOrCreateErrorMessageElement().textContent = message;
-      if (this.errorMessage?.style.display === 'none') {
-        this.errorMessage.style.display = '';
-      }
     } else if (this.errorMessage) {
-      this.errorMessage.style.display = 'none';
+      this.errorMessage.textContent = '';
```

Comment:
> This change needs a corresponding update in [`app/components/memorable_date_component.html.erb`](https://github.com/18F/identity-idp/blob/a10cd96f8f409d5b57249d1036c889d878439c39/app/components/memorable_date_component.html.erb#L122).
> 
> https://github.com/18F/identity-idp/blob/a10cd96f8f409d5b57249d1036c889d878439c39/app/components/memorable_date_component.html.erb#L122

Thanks for flagging! I'll update it.

Ideally we could avoid having the ValidatedField markup duplicated in MemorableDate, since I think it's prone to this sort of oversight happening again in the future.

Reply:
I'm planning to restore this behavior to fix the layout issues it was initially introduced for. In doing so, I noticed that the use of the inline style is problematic, since it will be blocked in production due to our content security policy.

Example error, in staging:

![image](https://user-images.githubusercontent.com/1779930/207890193-472078af-14c1-4aa3-bf77-b07d9e5d79f7.png)

(The local development experience of CSP being misleading is something I'd like to improve, tracked in LG-5354)

Reply:
Restored `display: none` in b86ab05.